client/modules/IDE/components/FileNode.jsx

```diff
@@ -303,6 +317,8 @@ class FileNode extends React.Component {
               className="sidebar__file-item-name"
               onClick={this.handleFileClick}
               data-testid="file-name"
+              onKeyDown={this.handleOnKeyboardDelete}
+              tabIndex={0}
```

Comment:
You shouldn't need `tabIndex` on a `button` because buttons are focusable by default.

Reply:
Oh sorry! Actually I was doing some experiments, forgetting to remove the line. 

Let me just update.